### PR TITLE
add active, focus, & hover states to buttons

### PR DESCRIPTION
**Pull request by oceanclimb**
https://trello.com/c/u3lnjLUg/1-bloom-add-button-hover-focus-active-state-styles

AC: add active, focus, & hover states to buttons. 

screen shot 2015-04-04 at 6 37 10 pm

screen shot 2015-04-04 at 6 37 13 pm

### CODE REVIEW

**Comment by oceanclimb** [comment]
Did you want the active & focus state for buttons @msisto ?

**Comment by oceanclimb** [comment]
Would we ever want the responsive block buttons that we use in the patient app in bloom?

File: elements/_button.scss [hunk 1]
```diff
@@ -18,11 +18,19 @@
 %button__style {
   background-color: $color--gray;
   color: $base-font-color;
+
+  &:hover {
+    background-color: shade($color--gray, 5%);
```
Comment (msisto) [inline elements/_button.scss]: Use `darken()`

**Comment by msisto** [comment]
@oceanclimb  Probably re: responsive blocks but let's wait until there is a reason to add it. That might make sense once we include Bloom in the patient app properly (or if we need it in Chirp for example)

**Comment by msisto** [comment]
@oceanclimb The card says add `:active` and `:focus` states, so yes.

**Comment by oceanclimb** [comment]
It seems like you have some specific ideas for the focus state with some type of affordance, Matt. So I'll let you write that code. @msisto

**Comment by oceanclimb** [comment]
http://24ways.org/2009/dont-lose-your-focus/

**Comment by msisto** [comment]
I am not sure what you are asking. Let's talk Monday.

**Comment by oceanclimb** [comment]
Yeah lets talk on Monday. Thanks.

**Comment by oceanclimb** [comment]
@msisto the styles are updated & simplified.

**Pull request closed without merging**